libclambcc/ClamBCAnalyzer/ClamBCAnalyzer.h

```diff
@@ -76,6 +82,7 @@ class ClamBCAnalyzer : public llvm::ModulePass
     unsigned maxGlobal = 0;
     std::vector<llvm::Constant *> globalInits;
     std::vector<const llvm::MDNode *> mds;
+    bool WriteDI = false;
```

Comment:
The code in ClamBCWriter is a command line option.  I have never used it, but I am leaving the option in place in case someone has a build script somewhere that uses it.  I have removed the flag from the analyzer.
(I also marked the option as depracated).

Reply:
Raynor is under the impression that we others are using the clamav and the compiler in ways we're unaware of and that we shouldn't risk breaking things we don't know exist.

I used to be superstitious about not removing things that look incorrect or like dead code for fear I would break something.  I've since dropped that and desire to make these projects work correctly the way we envision.  If someone else has a secret need that we've inadvertently broken, they can speak up about it and use an older release until it can be  fixed, and they should submit a fix themselves. 

That is, I don't recommend walking on eggshells. Just build and test for the known use cases so we can make it as clean as possible.

Reply:
works for me.  removed